scalableminds/webknossos
# Allow dollar in layer name

### URL of deployed dev instance (used for testing):
- https://allowdollarinlayernames.webknossos.xyz

### Steps to test:
- Open the dataset settings of any DS
- change one layer & the dataset name to include a $. Submitting should work (but likely running the DS in case a layer is renamed)
- Renaming a layer can be done via the advanced tab

### TODOs:
- [x] also allow `$` in `checkLayerNameForInvalidCharacters`
- [x] disallow `$` for datasetNames
- [x] test migration & reversion

### Issues:
- fixes https://scm.slack.com/archives/C5AKLAV0B/p1732724785230369
- fixes https://github.com/scalableminds/webknossos/issues/8172

------
(Please delete unneeded items, merge only when none are left open)
